### PR TITLE
added dynamic animator

### PR DESCRIPTION
**Pull request by RizanHa**
Added an optional dynamic animator to SCLAlertView. It can be configure by property "dynamicAnimatorActive" from SCLAppearance. The default animation is still like before.

### CODE REVIEW

File: SCLAlertView/SCLAlertView.swift [hunk 1]
```diff
@@ -781,21 +787,58 @@ open class SCLAlertView: UIViewController {
...
-            self.view.alpha = 1.0
-            self.baseView.center = animationCenter
-            }, completion: { finished in
-                UIView.animate(withDuration: animationDuration, animations: {
-                    self.view.alpha = 1.0
-                    self.baseView.center = rv.center
-                })
-        })
+        
+
```
Comment (vikmeup) [inline SCLAlertView/SCLAlertView.swift]: can you remove extra spaces in newly added code.

File: SCLAlertView/SCLAlertView.swift [hunk 2]
```diff
@@ -153,11 +153,13 @@ open class SCLAlertView: UIViewController {
         var contentViewCornerRadius : CGFloat
         var fieldCornerRadius : CGFloat
         var buttonCornerRadius : CGFloat
+        var dynamicAnimatorActive : Bool
+
```
Comment (vikmeup) [inline SCLAlertView/SCLAlertView.swift]: as well as here.

File: SCLAlertView/SCLAlertView.swift [hunk 3]
```diff
@@ -227,6 +229,9 @@ open class SCLAlertView: UIViewController {
     internal var buttons = [SCLButton]()
     fileprivate var selfReference: SCLAlertView?
     
+    
+
```
Comment (vikmeup) [inline SCLAlertView/SCLAlertView.swift]: and here.

File: Example/SCLAlertViewExample/ViewController.swift [hunk 4]
```diff
@@ -38,6 +38,7 @@ class ViewController: UIViewController {
 		_ = alert.addButton("Second Button") {
 			print("Second button tapped")
 		}
+
```
Comment (vikmeup) [inline Example/SCLAlertViewExample/ViewController.swift]: here as well.

**Comment by RizanHa** [comment]
done 👌

**Pull request merged**